### ISSUE TITLE
[Feature request] Query download(s) by tag

### ISSUE THREAD

**Issue by nthtrung09it** [tonyofrancis/Fetch#416]
Hey, thanks for the awesome library.
Currently I'm using tag to determine the download/request. (Because Download id is type of **Int**, but in my case we are using **String** for primary key, that is why I'm using tag).
So I need a method/fun to query download(s) by tag.

How about if we make Download **id** and **group_id** are type of **String** instead of **Int**?

Thanks

**Comment by tonyofrancis** [comment]
@nthtrung09it What we can do is add a method to get the downloads by tag.
`fetch.getDownloadsByTag(tag, callback)`.

**Comment by tonyofrancis** [comment]
@nthtrung09it This issue is now fixed in new Fetch version 3.0.10/Androidx 3.1.4.
```java
  fetch.getDownloadsByTag("Test", new Func<List<Download>>() {
            @Override
            public void call(@NotNull List<Download> result) {
                //get all downloads with the tag
            }
        });
```

**Comment by nthtrung09it** [comment]
Thanks @tonyofrancis, I've just checked and it works as expected.
